### DISCUSSION TITLE
error mastodon page

### DISCUSSION THREAD

**Discussion by NellyOfc** [mastodon/mastodon#24802] [General]
hello mastodn, someone from the team could help me, I recently installed mastodon, I did all the right steps and all mastodon services are online, but it stays on this white 
screen, do you know what can be for me to solve this?

![image](https://user-images.githubusercontent.com/87354471/235787213-7224ceda-8cfc-40cf-8d70-7d0f58c4c7c3.png)

**Comment by NellyOfc** [comment]
May 02 05:41:10 bh1 systemd[1]: Started mastodon-web.
May 02 05:41:13 bh1 bundle[33143]: [33143] Puma starting in cluster mode...
May 02 05:41:13 bh1 bundle[33143]: [33143] * Puma version: 5.6.5 (ruby 3.0.6-p2>
May 02 05:41:13 bh1 bundle[33143]: [33143] *  Min threads: 5
May 02 05:41:13 bh1 bundle[33143]: [33143] *  Max threads: 5
May 02 05:41:13 bh1 bundle[33143]: [33143] *  Environment: production
May 02 05:41:13 bh1 bundle[33143]: [33143] *   Master PID: 33143
May 02 05:41:13 bh1 bundle[33143]: [33143] *      Workers: 2
May 02 05:41:13 bh1 bundle[33143]: [33143] *     Restarts: (✔) hot (✖) phased
May 02 05:41:13 bh1 bundle[33143]: [33143] * Preloading application
May 02 05:41:33 bh1 bundle[33143]: [33143] * Listening on http://127.0.0.1:3000
May 02 05:41:33 bh1 bundle[33143]: [33143] Use Ctrl-C to stop
May 02 05:41:33 bh1 bundle[33143]: [33143] - Worker 0 (PID: 33198) booted in 0.>
May 02 05:41:33 bh1 bundle[33143]: [33143] - Worker 1 (PID: 33202) booted in 0.>
May 02 05:41:50 bh1 bundle[33202]: [a1b356b4-1690-427b-bfde-dd6c602c23f6] Chewy>
May 02 05:41:51 bh1 bundle[33202]: [a1b356b4-1690-427b-bfde-dd6c602c23f6] metho>
May 02 05:41:51 bh1 bundle[33198]: [37cead38-19d8-49b7-aae9-4a22a63ccc49] Chewy>
May 02 05:41:51 bh1 bundle[33198]: [f510ab15-6d01-445b-8b9f-9b30f760d61f] Chewy>
May 02 05:41:51 bh1 bundle[33198]: [eb575dcc-c044-457a-b7df-678fa25e0246] Chewy>
May 02 05:41:51 bh1 bundle[33202]: [98347092-fbaa-47bf-aa36-fbd89603b354] metho>
May 02 05:41:51 bh1 bundle[33202]: [abb46be2-b547-4497-bb03-a22c4afc9826] metho>
May 02 05:41:51 bh1 bundle[33202]: [6dc03560-90f0-4f93-8c71-05b4a3362300] metho>
May 02 05:41:51 bh1 bundle[33202]: [95c42031-99ed-4f6f-8b64-ba3184cfd169] metho>
lines 1-23...skipping...
May 02 05:41:10 bh1 systemd[1]: Started mastodon-web.
May 02 05:41:13 bh1 bundle[33143]: [33143] Puma starting in cluster mode...
May 02 05:41:13 bh1 bundle[33143]: [33143] * Puma version: 5.6.5 (ruby 3.0.6-p216) ("Birdie's Version")
May 02 05:41:13 bh1 bundle[33143]: [33143] *  Min threads: 5
May 02 05:41:13 bh1 bundle[33143]: [33143] *  Max threads: 5
May 02 05:41:13 bh1 bundle[33143]: [33143] *  Environment: production
May 02 05:41:13 bh1 bundle[33143]: [33143] *   Master PID: 33143
May 02 05:41:13 bh1 bundle[33143]: [33143] *      Workers: 2
May 02 05:41:13 bh1 bundle[33143]: [33143] *     Restarts: (✔) hot (✖) phased
May 02 05:41:13 bh1 bundle[33143]: [33143] * Preloading application
May 02 05:41:33 bh1 bundle[33143]: [33143] * Listening on http://127.0.0.1:3000
May 02 05:41:33 bh1 bundle[33143]: [33143] Use Ctrl-C to stop
May 02 05:41:33 bh1 bundle[33143]: [33143] - Worker 0 (PID: 33198) booted in 0.03s, phase: 0
May 02 05:41:33 bh1 bundle[33143]: [33143] - Worker 1 (PID: 33202) booted in 0.02s, phase: 0
May 02 05:41:50 bh1 bundle[33202]: [a1b356b4-1690-427b-bfde-dd6c602c23f6] Chewy request strategy is `mastodon`
May 02 05:41:51 bh1 bundle[33202]: [a1b356b4-1690-427b-bfde-dd6c602c23f6] method=GET path=/ format=html controller=HomeController action=index status=200 duration=383.>
May 02 05:41:51 bh1 bundle[33198]: [37cead38-19d8-49b7-aae9-4a22a63ccc49] Chewy request strategy is `mastodon`
May 02 05:41:51 bh1 bundle[33198]: [f510ab15-6d01-445b-8b9f-9b30f760d61f] Chewy request strategy is `mastodon`
May 02 05:41:51 bh1 bundle[33198]: [eb575dcc-c044-457a-b7df-678fa25e0246] Chewy request strategy is `mastodon`
May 02 05:41:51 bh1 bundle[33202]: [98347092-fbaa-47bf-aa36-fbd89603b354] method=GET path=/500.html format=*/* controller=ApplicationController action=raise_not_found >
May 02 05:41:51 bh1 bundle[33202]: [abb46be2-b547-4497-bb03-a22c4afc9826] method=GET path=/500.html format=*/* controller=ApplicationController action=raise_not_found >
May 02 05:41:51 bh1 bundle[33202]: [6dc03560-90f0-4f93-8c71-05b4a3362300] method=GET path=/inert.css format=css controller=ApplicationController action=raise_not_found>
May 02 05:41:51 bh1 bundle[33202]: [95c42031-99ed-4f6f-8b64-ba3184cfd169] method=GET path=/custom.css format=css controller=CustomCssController action=show status=200 >
May 02 05:41:51 bh1 bundle[33198]: [f510ab15-6d01-445b-8b9f-9b30f760d61f] method=GET path=/500.html format=*/* controller=ApplicationController action=raise_not_found >
May 02 05:41:51 bh1 bundle[33198]: [eb575dcc-c044-457a-b7df-678fa25e0246] method=GET path=/500.html format=html controller=ApplicationController action=raise_not_found>
May 02 05:41:51 bh1 bundle[33198]: [37cead38-19d8-49b7-aae9-4a22a63ccc49] method=GET path=/500.html format=html controller=ApplicationController action=raise_not_found>
May 02 05:41:51 bh1 bundle[33198]: [6a498d0d-2c51-4b5b-a873-ac3f79c4023d] method=GET path=/inert.css format=css controller=ApplicationController action=raise_not_found>
May 02 05:41:52 bh1 bundle[33198]: [41c2d28c-0533-40bf-864e-e12a0e10a6d0] method=GET path=/favicon.ico format=html controller=ApplicationController action=raise_not_fo>
May 02 05:41:52 bh1 bundle[33198]: [bb0772c8-6bce-4c8f-9139-a077b255a723] method=GET path=/500.html format=html controller=ApplicationController action=raise_not_found>
May 02 05:41:52 bh1 bundle[33202]: [d76f799e-7882-4106-97a2-66f0800b4d98] method=GET path=/manifest format=json controller=ManifestsController action=show status=200 d>
May 02 05:41:52 bh1 bundle[33202]: [edffd2b3-7314-4064-b42b-dfc98cf5054e] method=GET path=/500.html format=html controller=ApplicationController action=raise_not_found>
May 02 05:41:53 bh1 bundle[33202]: [b6d75f07-1d54-4154-af12-ad78dea2973d] method=GET path=/500.html format=html controller=ApplicationController action=raise_not_found>
May 02 05:41:53 bh1 bundle[33202]: [61bb02b8-cd29-45c7-8199-8430e22e6fe1] method=GET path=/500.html format=html controller=ApplicationController action=raise_not_found>
May 02 05:41:54 bh1 bundle[33202]: [c970c71e-7fbd-4b57-9e8d-e1d351b46c87] method=GET path=/ format=html controller=HomeController action=index status=200 duration=49.2>
May 02 05:41:54 bh1 bundle[33198]: [28e0466a-7dfa-42b0-a1a6-fdea73a66d49] method=GET path=/inert.css format=css controller=ApplicationController action=raise_not_found>
May 02 05:41:54 bh1 bundle[33198]: [3013cc06-20dc-474f-a1c0-c8d38e7ee6db] method=GET path=/500.html format=*/* controller=ApplicationController action=raise_not_found >
May 02 05:41:54 bh1 bundle[33202]: [e3978634-484c-4d08-8519-8e5bda72cf10] method=GET path=/500.html format=*/* controller=ApplicationController action=raise_not_found >
May 02 05:41:54 bh1 bundle[33202]: [002f6225-72f1-49e9-b86a-6c71f071c289] method=GET path=/500.html format=html controller=ApplicationController action=raise_not_found>
May 02 05:41:54 bh1 bundle[33202]: [8a1f2462-1194-4625-acff-8b0e478ced91] method=GET path=/500.html format=*/* controller=ApplicationController action=raise_not_found >
May 02 05:41:54 bh1 bundle[33202]: [92b2ed18-27c9-4abb-9e71-f352f5095c4c] method=GET path=/500.html format=html controller=ApplicationController action=raise_not_found>
May 02 05:41:55 bh1 bundle[33198]: [70c55a27-ad14-4bed-9f8c-1572b811432b] method=GET path=/inert.css format=css controller=ApplicationController action=raise_not_found>
May 02 05:41:55 bh1 bundle[33198]: [11202ffc-99de-44d4-85d3-0b13d6ce1c31] method=GET path=/500.html format=html controller=ApplicationController action=raise_not_found

**Comment by renchap** [comment]
It looks like your webserver is preventing assets to be loaded, the `/inert.css` queries should not return a not found exception like in your log.

If you followed the official documentation, then your nginx config is probably incorrect (see https://docs.joinmastodon.org/admin/install/#setting-up-nginx)

**Comment by suuuuugao** [comment]
I got the same error too.
But I was able to improve it by doing the following.
```
# nano /etc/nginx/nginx.conf
user mastodon;
※user www-data change 

# mkdir -p /var/nginx/cache/
# chown -R mastodon:mastodon /var/nginx/cache
# chown -R mastodon:mastodon /var/lib/nginx

# nginx -t
nginx: the configuration file /etc/nginx/nginx.conf syntax is ok
nginx: configuration file /etc/nginx/nginx.conf test is successful
# systemctl reload nginx
```
References
https://pc.watch.impress.co.jp/docs/column/nishikawa/1468283.html

My environment is Ubuntu22 non-Docker
I hope this will be a solution.
